src/magic_folder/client.py

```diff
@@ -181,8 +199,7 @@ def _authorized_request(self, method, url, body=b""):
             )
 
         body = yield _get_content_check_code([http.OK, http.CREATED], response)
-        # all responses should contain JSON
-        returnValue(json.loads(body))
+        returnValue(body)
```

Comment:
now we won't validate that it's actually-json?

Reply:
I've changed `_get_content_check_code` above to return JSON as parsed by treq (largely because I wanted to pass the parsed version to `MagicFolderApiError` in the error case.

Reply:
I've updated the name of the function to make that clear.